katib-v1alpha2/katib-manager/overlays/application/application.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: katib-manager 
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: katib-manager 
+      app.kubernetes.io/instance: katib-manager 
+      app.kubernetes.io/managed-by: kfctl
+      app.kubernetes.io/component: katib
+      app.kubernetes.io/part-of: kubeflow
+      app.kubernetes.io/version: v0.6
+  componentKinds:
+  - group: core
+    kind: Service
+  - group: apps
+    kind: Deployment
+  descriptor:
+    type: "katib-manager"
+    version: "v1alpha2"
+    description: "Katib manager allows users to launch AutoML jobs on katib and manage them."
```

Comment:
Katib manager doesn't launch jobs; it is more like a storage API layer.

Reply:
I changed the description.